Makefile

```diff
@@ -1016,6 +1016,12 @@ debug_api_docs: $(FUZION_BASE) $(CLASS_FILES_TOOLS_DOCS)
 unicode: $(BUILD_DIR)/UnicodeData.java
 	cp $^ $(SRC)/dev/flang/util/UnicodeData.java
 
+# regenerate $(FZ_SRC_LIB)/unicode/data.fz using the latest UnicodeData.txt.
+.phony: lib/unicode/data
+lib/unicode/data: unicode $(CLASS_FILES_UTIL_UNICODE)
```

Comment:
I think this should not be a phony target and not depend on the phony target  `unicode`. 

Instead, this should better be a normal target for `$(BUILD_DIR)/generated/lib/unicode/data.fz` (or similar). Then, the rule to copy this over to `$(FZ_SRC_LIB)/lib/unicode/data.fz` should mabe be part of the phony target `unicode`.

The result will be that whenever a new unicode data is published, we can run the phony target  `unicode` once and everything needed will be updated.